local_pathfinding/local_path.py

```diff
@@ -37,6 +64,9 @@ def update_if_needed(
         global_path: GlobalPath,
         filtered_wind_sensor: WindSensor,
     ):
+        """Updates the LocalPath with an updated OMPLPath and new local waypoints.
```

Comment:
I think you forgot to document the arguments. You can also write when the path is updated. Currently, it is when a new path is found

Reply:
Isn't that the same as saying the following from line 32-34 (I have made edits since the review):
            `gps` (GPS): Data from the GPS sensors.
            `ais_ships` (AISShips): Data from the AIS receiver.
            `global_path` (GlobalPath): Data from the Globalpath server.
            `filtered_wind_sensor` (WindSensor): Data from the windsensors.